hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSDirectory.java

```diff
@@ -1475,6 +1486,26 @@ void addYieldCount(long value) {
   public INodeMap getINodeMap() {
     return inodeMap;
   }
+  public GSet<INode, INodeWithAdditionalFields> getTempINodeMap() {
```

Comment:
Inconsistent naming. Getters should be named getMememberName().

Reply:
removed this function, as it is not used at all...